examples_tests/60.ClusteredRendering/cull_common.glsl

```diff
@@ -0,0 +1,247 @@
+#ifndef CULL_COMMON_H
```

Comment:
put in `nbl/builtin/glsl/ext/clusteredlighting/` and add appropriate `#ifdef/#define` guards and `nbl_glsl_ext_ClusteredLighting_` prefixes to declarations and variable names